config/config_test.go

```diff
@@ -63,14 +65,31 @@ func TestInvalidConfigs(t *testing.T) {
 	for k, v := range invalidConfigs {
 		t.Run(k, func(t *testing.T) {
 			filename := mkConfig(t, v)
-			_, err := ReadConfigFile(filename, "", "")
+			_, err := ReadConfigFile(filename, "", "", "")
 			if err == nil {
 				t.Fatal("this config must generate an error")
 			}
 		})
 	}
 }
 
+func TestConfigDir(t *testing.T) {
+	colorGreen := "\033[32m"
+	colorReset := "\033[0m"
+
+	cwd, _ := os.Getwd()
```

Comment:
errors not really relevant to the test, but imo I like to `panic(err)` any such errors. Might save someone some work debugging in the future.

Reply:
or t.Fatal(err)

Reply:
sorry this was me troubleshooting my test :grimacing: